common-lib/common/g.libsonnet

```diff
@@ -0,0 +1 @@
+import 'github.com/grafana/grafonnet/gen/grafonnet-latest/main.libsonnet'
```

Comment:
I would suggest pinning to a specific Grafana version and let Grafana handle upgrading.